gazebo/rendering/Visual.cc

```diff
@@ -817,7 +817,9 @@ void Visual::SetScale(const ignition::math::Vector3d &_scale)
   {
     this->dataPtr->sceneNode->setScale(
         Conversions::Convert(this->dataPtr->scale));
-  } else {
+  }
+  else
+  {
     gzerr << Name() << ": Size of the collision contains one or several zeros." <<
```

Comment:
I also get an error here on this fine for going over 80 chars

Reply:
Ouch, sorry, now a new PR will need to be made for this, my bad.